client/src/components/Console.tsx

```diff
@@ -1,117 +1,195 @@
-                        </Tab>
-                        <Tab>
-                            Timeline
-                        </Tab>
-                    </TabList>
-                </TabGroup>
-            </div>
-            <Grid numItemsLg={6} className="gap-6 mt-6">
-                <Col numColSpanLg={5}>
-                    <Card className="h-full p-0 m-0" style={{ margin: 0, padding: 0 }}>
-                        {renderCurrentSelection(selectedView)}
-                    </Card>
-                </Col>
+  useEffect(() => {
+    const fetchDocuments = async () => {
+      try {
+        const response = await API.getAllKxDocuments();
+        setDocuments(response);
 
-                <Col numColSpanLg={1}>
-                    <div className="space-y-6">
-                        <FormDialog />
+        let count = 0;
+        const filteredDocuments = response.filter((doc) => {
+          if (doc.doc_coordinates?.type === "EntireMunicipality") {
+            count++;
+            return false;
+          }
+          return true;
+        });
 
-                        {/* <Card>
-                            <Text>Select</Text>
-                            <Select
-                                className="mt-2" value={groupKey} onValueChange={setGroupKey}
-                            >
-                                <SelectItem value="1">
-                                    1
-                                </SelectItem>
-                                <SelectItem value="2">
-                                    2
-                                </SelectItem>
-                                <SelectItem value="3">
-                                    3
-                                </SelectItem>
-                                <SelectItem value="4">
-                                    4
-                                </SelectItem>
-                                <SelectItem value="5">
-                                    5
-                                </SelectItem>
-                            </Select>
-                        </Card> */}
-                        <Card>
-                            <Metric>KIRUNA</Metric>
-                            <Title>How to move a city</Title>
-                            <Subtitle>Mapping the process of Kiruna relocation.</Subtitle>
-                            <Text>
-                                Kiruna, a Swedish city, is being relocated in phases since 2010 to prevent damage from mining activities.
-                            </Text>
-                        </Card>
-                        <Card className="hidden lg:block w-full h-40">
-                            <img src="/kiruna.png" alt="Kiruna" className="w-full h-full object-contain" />
-                        </Card>
-                    </div>
-                </Col>
+        // Find the document which assigned to a point
+        const documentsAssignedToPoints = response.filter((doc) => {
```

Comment:
you are not using this filter.anyway i already filtered for drawable things or entire municipality so dont need this filter